elastic/kibana
# [ResponseOps][Alerts] Update stack alerts page feature labels and filters

## Summary

Updates the alerts table feature cell renderers and feature filters according to #183615.

## To verify

1. Create some Stack, Stack Monitoring, and ML rules that fire alerts
2. Navigate to `Stack Management` > `Alerts`
3. Verify that the feature quick filter (found in the <img width="22" alt="image" src="https://github.com/elastic/kibana/assets/18363145/847c3fc4-1c23-4846-ae32-d66b10de7ad5"> menu of the KQL bar) respects the changes outlined in points 1 and 2 of #183615
4. Verify that the alerts from Stack rules show a `Stack` label in the table
6. Verify that the alerts from Stack Monitoring rules show a `Stack Monitoring` label in the table
7. Verify that filtering by `Stack rule types` shows only Stack and ML alerts in the table
8. Verify that filtering by `Observability rule types` doesn't show Stack Monitoring alerts
9. Verify that filtering by `Stack Monitoring rule types` only shows Stack Monitoring alerts
